### PR TITLE
Tailwind css

### PR DESCRIPTION
**Pull request by aleciabenjamin**
[Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2315151/stories/164246410)

This branch includes styling to the "Landing", "Sign up" and "Log in" views.  
screenshot 2019-03-02 at 19 44 28
screenshot 2019-03-02 at 19 45 11
screenshot 2019-03-02 at 19 45 38

### CODE REVIEW

**Review by yanqiuzhang — APPROVED** [review]

**Review by NiciP — APPROVED** [review]

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -10,7 +10,18 @@
...
     %p= notice
     %p= alert
-    = yield
\ No newline at end of file
+  = yield
+  %footer
+    %h2 Contact Us
+    %p Tel. 555 6677
+    %p email: www.crafty@clean.se  
+
```
Comment (NiciP) [inline app/views/layouts/application.html.haml]: remove extra line pls

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -17,10 +17,12 @@
       <%= f.label :remember_me %>
     </div>
   <% end %>
+
```
Comment (NiciP) [inline app/views/devise/sessions/new.html.erb]: remove extra line pls

Comment (NiciP) [inline app/views/devise/sessions/new.html.erb]: remove extra line pls

**Review by NiciP — CHANGES_REQUESTED** [review]
Just remove 3 blank lines as below pls

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  default_url_options :host => "example.com"
```
Comment (faraznaeem) [inline config/routes.rb:2]: What is this route?

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Comment by faraznaeem** [comment]
Why isn't semaphore kicking in?

File: config/routes.rb (hunk 3 above)
Comment (aleciabenjamin) [reply config/routes.rb:2]: When clicking on 'send a new password to this email' link we encountered a host/path issue.  This fixed it.

Comment (aleciabenjamin) [reply config/routes.rb:2]: Is there a different fix you'd like us to use?  Build has passed on semaphore.

**Review by tochman — APPROVED** [review]

**Pull request merged**